website/source/docs/enterprise/performance-standby/index.html.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: "docs"
+page_title: "Performance Standby Nodes - Vault Enterprise"
+sidebar_current: "docs-vault-enterprise-perf-standbys"
+description: |-
+  Performance Standby Nodes - Vault Enterprise
+---
+
+# Performance Standby Nodes 
+
+Vault supports a multi-server mode for high availability. This mode protects
+against outages by running multiple Vault servers. High availability mode
+is automatically enabled when using a data store that supports it. You can 
+learn more about HA mode on the [Concepts](/docs/concepts/ha.html) page.
+
+Vault Enterprise offers additional features that allow HA nodes to service
+read-only requests on the local standby node. Read-only requests are requests
+that do not modify Vault's storage. 
+
+## Server-to-Server Communication
+
+Performance Standbys require the request forwarding method described in the [HA
+Server-to-Server](/docs/concepts/ha.html#server-to-server-communication) docs.
+
+A performance standby will connect to the active node over the existing request
+forwarding connection. If selected by the active node to be promoted to a
+performance standby it will be handed a newly-generated private key and certificate
+for use in creating a new mutually-authenticated TLS connection to the cluster
+port. This connection will be used to send updates from the active node to the
+standby.
+
+## Request Forwarding
+
+A Performance Standby will attempt to process requests that come in. If a
+storage write is detected the standby will forward the request over the cluster
+port connection to the active node. If the request is read-only the Performance
+Standby will handle the requests locally.
```

Comment:
May want to mention that this means that sending requests to Perf Stanbys that result in writes will be slightly slower than going directly to the active node, so it's ideal if clients can separate requests out based on advance knowledge of the behavior of the call.